pkg/reconciler/broker/broker.go

```diff
@@ -206,12 +209,36 @@ func (r *Reconciler) reconcileBroker(ctx context.Context, b *brokerv1beta1.Broke
 			m.SetState(config.State_UNKNOWN)
 		}
 	})
+	// Insert each Trigger to the config.
+	for _, t := range triggers {
```

Comment:
This loop should be done inside the broker mutation function. Your current approach will be more expensive on memory.